### PR TITLE
Fixing force flag + Updating required packages

### LINKED ISSUE #951: Force flag does not work to force removal of LUIS application.
## Tool
Name: LUIS
Version: 2.2.0
OS: Debian stretch

## Describe the bug
Trying to delete my LUIS applications and restart the ```botbuilder-samples/samples/csharp_dotnetcore/14.nlp-with-dispatch``` sample from scratch. I delete the Dispatch application from [luis.ai/applications](https://www.luis.ai/applications) without problems. When I try to delete Weather or Home Automation application from the website I get the helpful "BadArgument: The application Weather is referenced by a Dispatch application. Please, ask the application owner to remove the referenece, your use the flag force=true to force delete the relation." of course there is no way to set the force flag from the web UI so I try using the LUIS command line.

## To Reproduce
1. run the following command to find you appId
```bash
$ luis list applications --authoringKey <luisAuthoringKey>
```
2. run the command to delete the application
```bash
$ luis delete application --appId <appId>  --authoringKey <luisAuthoringKey>
```
see error "The application Weather is referenced by a Dispatch application. Please, ask the application owner to remove the referenece, your use the flag force=true to force delete the relation."

3. add force parameter
```bash
$ luis delete application --appId <appId>  --authoringKey <luisAuthoringKey> --force
```

## Expected behavior
Success

## Actual behavior
The application Weather is referenced by a Dispatch application. Please, ask the application owner to remove the referenece, your use the flag force=true to force delete the relation.

### PR DESCRIPTION
**Pull request by kayousef**
Fixes #951

## Proposed Changes
  - Fix the force flag used with the delete API. Currently the delete API call fails if the force flag is used
  - Update Packages file with the required version of typescript to make the first installation easier for new contributors

### CODE REVIEW

File: packages/LUIS/package.json [hunk 1]
```diff
@@ -61,11 +61,11 @@
     "window-size": "^1.1.0"
   },
   "devDependencies": {
-    "typescript": "^3.1.1",
     "enum-files": "^1.1.0",
+    "mocha": "^5.2.0",
```
Comment (cleemullins) [inline packages/LUIS/package.json]: Why the changes to the package.json? 

Can you revert these?

File: packages/LUIS/bin/luis.js [hunk 2]
```diff
@@ -265,7 +265,7 @@ async function runProgram() {
                                 return;
                             }
                         }
-                        result = await client.apps.deleteMethod(args.region, args.cloud, args.appId, args.force, args);
+                        result = await client.apps.deleteMethod(args.region, args.cloud, args.appId, { force: args.force }, args);
```
Comment (cleemullins) [inline packages/LUIS/bin/luis.js]: Code change should have a corresponding test change.

Comment (kayousef) [reply packages/LUIS/bin/luis.js]: This was not working at all. Can you advise on where to find the original test to fix it?

File: packages/LUIS/package.json (hunk 1 above)
Comment (kayousef) [reply packages/LUIS/package.json]: I spent some time to get the tool to work, becaus it required a specific version of typescript. So, I though I might update the packages file to make everyone else's life easier going forward.

File: packages/LUIS/bin/luis.js (hunk 2 above)
Comment (cleemullins) [reply packages/LUIS/bin/luis.js]: It wasn't working because there was no test. :)

File: packages/LUIS/package.json (hunk 1 above)
Comment (cleemullins) [reply packages/LUIS/package.json]: That's great. Can you update the PR description to include the? Otherwise it looks like an accident.

Comment (kayousef) [reply packages/LUIS/package.json]: It's already in both title and description. Let me know if you think of a better wording

**Comment by cleemullins** [comment]
I approved and signed off 2+ hours ago! :) Merge when your ready...
